databrowser/src/components/quickview/QuickViewFullscreenGallery.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="gallery-ct">
+    <div
+      v-for="(image, i) in images"
+      :id="`image-in-gallery-${i}`"
+      :key="i"
+      class="shrink-0 w-full bg-center bg-cover"
+      :style="{
```

Comment:
Tailwind cannot be used to pass dynamic background urls fetched from APIs.

Reply:
@MatteoBiasi you're right, I thought Tailwind generates the appropriate classes on-the-fly but it does so only for classes that it knows during compile time - which makes sense, otherwise it had to include some dynamic compile mechanism in the final output. Never mind then :+1:

Reply:
I've found the issue: when started, Vite advises 127.0.0.1:3000 as the local url for the project instead of localhost:3000. Even though 127.0.0.1 should be equal to localhost, there's probably something within the project that doesn't resolve the hostname correctly and prevents the requests to load if it is 127.0.0.1. Therefore I suggest to change the default host to localhost instead of 127.0.0.1, which should be the default with the latest versions of Vite: https://github.com/vitejs/vite/pull/8543

Reply:
Hi @MatteoBiasi, nice finding, well done 👍

Is it correct that this problem affects the local development environment only (host resolution problem)? If so, what is your environment (os, node version, ...) and how do you start the dev server?

What do you think, is it better to provide a separate PR with the appropriate settings or would you like to add the change to your branch such that it gets more obvious why / where it was necessary to make the change?